server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -369,8 +369,7 @@ private void validateServerFamily(ArrayList<Short> violations, ColumnUpdate colu
   private void validateSuspendLocationFamily(ArrayList<Short> violations,
       ColumnUpdate columnUpdate) {
     String qualStr = new String(columnUpdate.getColumnQualifier(), UTF_8);
-    String suspendColQualStr =
-        new String(SuspendLocationColumn.SUSPEND_COLUMN.getColumnQualifier().getBytes(), UTF_8);
+    String suspendColQualStr = SuspendLocationColumn.SUSPEND_COLUMN.getColumnQualifier().toString();
```

Comment:
We could make a constant for this String.  For this case do not need to keep recomputing the string.  Could add that constant to the SuspendLocationColumn.

Reply:
Added. Also checked the other columns in MetadataSchema; SuspendLocationColumn was the only one that wasn't consistent